app/controllers/request_purpose.js

```diff
@@ -49,7 +49,7 @@ export default Ember.Controller.extend(cancelOrder, {
 
   isOnlineOrder() {
     return (
-      this.getSelectedBookingTypeId() ===
+      this.getSelectedBookingTypeId() ==
```

Comment:
Why type check was removed? I'd prefer to convert string to integer rather than removing type check. 
This is fine but for sake of consistency use `===` and convert string to integer using `+`;